Core/Main/itkElastixRegistrationMethod.hxx

```diff
@@ -847,7 +847,7 @@ ElastixRegistrationMethod<TFixedImage, TMovingImage>::GetNthTransform(const unsi
 
 template <typename TFixedImage, typename TMovingImage>
 auto
-ElastixRegistrationMethod<TFixedImage, TMovingImage>::GetCombinationTransform() const -> SmartPointer<TransformType>
+ElastixRegistrationMethod<TFixedImage, TMovingImage>::GetCombinationTransform() const -> TransformType *
```

Comment:
OK, LFTM to replace the smart pointer by a raw pointer here, because the "CombinationTransform" is still owned (and kept alive) by ElastixRegistrationMethod. Thanks again Matt.

Reply:
In this particular case (specifically for `GetCombinationTransform()`), I agree that returning a raw pointer should be fine. Because the "CombinationTransform" is still owned by the ElastixRegistrationMethod object anyway. Shall I make a separate PR to just do this adjustment for `GetCombinationTransform()`?